sources/ClangSharp/Internals/NativeTypeNameAttribute.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Diagnostics;
+
+namespace ClangSharp
+{
+    [Conditional("DEBUG")]
```

Comment:
`Conditional("DEBUG")` will cause the attribute to be stripped from release binaries, avoiding bloat.